doc/source/cluster/key-concepts.rst

```diff
@@ -60,19 +60,31 @@ sit idle, the autoscaler will remove worker nodes from the cluster.
 It is important to understand that the autoscaler only reacts to task and actor resource requests, and not application metrics or physical resource utilization.
 To learn more about autoscaling, refer to the user guides for Ray clusters on :ref:`VMs <cloud-vm-index>` and :ref:`Kubernetes <kuberay-index>`.
 
+
+.. _ray_driver:
+
+Ray Driver
+----------
+
+A Ray driver is the process that calls the Ray tasks and actors in the user application.
+A driver is started by running `ray.init()`. The script that calls `ray.init()` may also be referred to as the "driver script".
```

Comment:
I think it would be better to define Ray driver in the same subsection as "Ray job", since we are trying to elevate jobs not drivers.

Also, it is not exactly accurate that the driver needs to call `ray.init()`, since this is now called automatically for certain Ray APIs, like f.remote().

Reply:
The glossary draft definitions is not correct. Will fix it.